machine_learning/src/face_detect_model/data/detectFaceAndClip.py

```diff
@@ -1,97 +1,176 @@
 import cv2
 import os
 import yaml
+import numpy as np
+import argparse
+import logging
+from dotenv import load_dotenv
 
 
-def load_cascade(cascade_path):
-    """Haar Cascadeを読み込む"""
-    return cv2.CascadeClassifier(cv2.data.haarcascades + cascade_path)
+from detectFaceUtil import (
+    load_cascade,
+    detect_face,
+    clip_and_resize_face,
+    save_face,
+)
 
+load_dotenv()
 
-def detect_face(
-    image, face_cascade, scaleFactor=1.1, minNeighbors=15, minSize=(50, 50)
-):
-    """画像から顔を検出する"""
-    gray = cv2.cvtColor(image, cv2.COLOR_BGR2GRAY)
-    faces = face_cascade.detectMultiScale(
-        gray, scaleFactor=scaleFactor, minNeighbors=minNeighbors, minSize=minSize
-    )
-    return faces
-
-
-def clip_and_resize_face(face, image, image_size):
-    """検出された顔をクリップし、指定サイズにリサイズする"""
-    (x, y, w, h) = face
-    face_roi = image[y : y + h, x : x + w]
-    resized_face = cv2.resize(face_roi, image_size, interpolation=cv2.INTER_AREA)
-    return resized_face
+logging.basicConfig(
+    format="%(asctime)s - %(levelname)s - %(name)s - PID: %(process)d -  %(message)s",
+    datefmt="%m/%d/%Y %H:%M:%S",
+    level=logging.INFO,
+)
+logger = logging.getLogger(__name__)
 
 
-def save_face(face, save_dir, save_file_name):
-    """クリップされた顔画像を保存する"""
-    os.makedirs(save_dir, exist_ok=True)
-
-    save_path = os.path.join(save_dir, save_file_name)
-    cv2.imwrite(save_path, face)
+def load_image(args, blobs=None):
+    # 画像の読み込み
+    images = []
+    if args.env == "local":
+        for image_path in os.listdir(args.image_dir_path):
+            logger.info(f"loading: {image_path}")
+            image = cv2.imread(args.image_dir_path + image_path)
```

Comment:
```suggestion
            image = cv2.imread(os.path.join(args.img_dir_path, image_path))
```
実際にコマンドライン引数で受け取るオプションと絶妙に変数名が違うのは保守性に欠ける気がします

Reply:
タイポでした😭